packages/devtools/web/main.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:devtools/src/framework/framework_core.dart';
 import 'package:devtools/src/main.dart';
+import 'package:platform_detect/platform_detect.dart';
```

Comment:
Good call finding a package that already does this!

Reply:
that package is maintained by a company that is a long time dart customer so is quite safe for us to use. if the package was maintained by someone random user we would need to be more careful.